lib/saluki-core/src/data_model/event/log/mod.rs

```diff
@@ -0,0 +1,85 @@
+//! Logs.
+
+use std::num::NonZeroU64;
+
+use saluki_context::tags::SharedTagSet;
+use stringtheory::MetaString;
+
+/// Internal representation of a log event aligned to Datadog logs payload fields https://docs.datadoghq.com/api/latest/logs/#log-object.
+#[derive(Clone, Debug, PartialEq)]
+pub struct Log {
+    /// Log message body.
+    message: MetaString,
+    /// Log status/severity (e.g., "info", "warn", "error").
+    status: Option<MetaString>,
```

Comment:
Would it be better to use an enum for the status?

Reply:
My question as well, unless this is specifically allowed to be free-form to cope with non-standard log levels.